static/sass/custom/_series_tags.scss

```diff
@@ -0,0 +1,20 @@
+.series-tag {
+  display: inline-block;
+  border: 1px solid $color-mid-dark;
+  border-radius: 4px;
+  font-size: 0.875rem;
+  font-weight: 500;
+  color: $color-mid-dark;
+  padding: 0 0.5rem;
+  margin-right: 0.25rem;
+  text-transform: capitalize;
+
+  &.kubernetes {
```

Comment:
Could you BEM this class. Making Kubernetes a modifier and extending the series-tag placeholder.